### PR TITLE
Added Input component with an icon

### PR DESCRIPTION
**Pull request by GeekaN2**
Simple Input component with size props and an icon.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "vite": "^3.0.4"
   },
   "dependencies": {
+    "@codexteam/icons": "^0.0.2",
```
Comment (ilyamore88) [inline package.json]: You don't use it. So, remove pls.

File: src/commons/components/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+export const Input: React.FC<InputProps & React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>> = ({
+    size = 'm',
+    iconLeft,
+    className,
+    containerClassName,
+    ...props
+}) => {
+    console.log(styles);
```
Comment (slaveeks) [inline src/commons/components/Input/Input.tsx]: А зачем тут лог?

File: src/commons/components/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import cn from 'classnames';
+
+export interface InputProps {
+    size?: 's' | 'm' | 'l';
+    iconLeft?: React.ReactNode;
+    className: string;
+    containerClassName?: string;
+}
+
+export const Input: React.FC<InputProps & React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>> = ({
```
Comment (ilyamore88) [inline src/commons/components/Input/Input.tsx]: Could you please move this type in generic to something like `type PropsWithHTMLInput = ...`?

File: src/commons/components/Input/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    iconLeft,
+    className,
+    containerClassName,
+    ...props
+}) => {
+    return (
+        <span className={cn(styles.container, containerClassName)}>
+            <span className={styles.iconLeft}>
+                {Boolean(iconLeft) && iconLeft}
+            </span>
```
Comment (ilyamore88) [inline src/commons/components/Input/Input.tsx]: Don't render this span if there is not an icon.

**Review by ilyamore88 — APPROVED** [review]

**Pull request merged**